tap_purecloud/__init__.py

```diff
@@ -577,10 +580,27 @@ def sync_conversations(api_client: ApiClient, config):
                                 singer.write_schema(
                                     'conversation_participant_session_segment',
                                     schemas.conversation_participant_session_segment,
-                                    ['conversation_id', 'participant_id', 'session_id', 'segment_end']
+                                    ['conversation_id', 'participant_id', 'session_id', 'segment_start']
```

Comment:
`segment_end` can be null, but `segment_start` cannot..? Breaks the pkey requirement. Unfortunately there's no `segment_id` and the docs say that `segment_start` is optional (though every field from every response type is optional) https://developer.genesys.cloud/devapps/sdk/docexplorer/purecloudpython/AnalyticsConversationSegment